src/codegen/object.rs

```diff
@@ -305,4 +390,17 @@ pub fn generate_reexports(
         }
         traits.push(format!("\tpub use super::{};", analysis.trait_name));
     }
+
+    let mut generate_builder = false;
+    for property in &analysis.properties {
+        if (!property.is_get && property.construct) || property.construct_only {
+            generate_builder = true;
+            break;
```

Comment:
This not different condition from that used in `generate`?
Maybe better define separate function here?

Reply:
This function can return iterator filtered by `(!property.is_get && property.construct) || property.construct_only`,
and it can be used in all 3 places